components/p3a/brave_p3a_log_store.cc

```diff
@@ -24,7 +24,7 @@ void RecordP3A(uint64_t answers_count) {
   int answer = 0;
   if (1 <= answers_count && answers_count < 5) {
     answer = 1;
-  } else if (5 <- answers_count && answers_count < 10) {
+  } else if (5 <= answers_count && answers_count < 10) {
```

Comment:
`else if (5 < answers_count && answers_count < 10) {`

Shouldn't the implementation be like this? If we use `<=` both 1 and 2 response values will be valid and which one will be sent?

Reply:
We'd want 5 to be considered a proper answer too; we could also change to be `4 < answers_count && anwers_count < 10`